src/Core/Components/Icons/FluentIcon.razor.cs

```diff
@@ -19,7 +19,7 @@ public partial class FluentIcon<Icon> : FluentComponentBase
 
     /// <summary />
     protected string? StyleValue => new StyleBuilder(Style)
-        .AddStyle("width", Width ?? $"{_icon.Width}px")
+        .AddStyle("width", Width ?? $"{_icon.Width}px", Width != string.Empty)
```

Comment:
In line with @dvoituron's comment, probably better to check for IsNullOrWhitespace() ?

Reply:
That is what you get when doing a coffee review on a plane 😂